docs/CHANGELOG.asciidoc

```diff
@@ -10,11 +10,302 @@
+*Details* +
+For package policy upgrades, the packagePolicy `upgrade` endpoint format supports a mutative upgrade operation (when `dryRun: false`) and a read-only dry run operation (when `dryRun: true`):
+
+[source,text]
+--
+ POST /package_policies/upgrade
+ {
+   packagePolicyIds: [...],
+   dryRun: false
+ }
+--
+
+For more information, refer to {kibana-pull}118854[#118854].
+
+*Impact* +
+The endpoint is now split into two separate endpoints:
+
+[source,text]
+--
+ POST /package_policies/upgrade
+ {
+   packagePolicyIds: [...]
+ }
+
+ POST /package_policies/upgrade/dry_run
+ {
+   packagePolicyIds: [...]
+ }
+--
+====
+      
+[discrete]
+[[breaking-119945]]
+.Removes APM jobs from Machine Learning
+[%collapsible]
+====
+*Details* +
+APM NodeJS and RUM Javascript anomaly detection job modules have been removed. For more information, refer to {kibana-pull}119945[#119945].
+
+*Impact* +
+When you upgrade to 8.0.0, you are unable to create and view the APM NodeJS and RUM Javascript jobs in Machine Learning.
+====
+
+[discrete]
+[[breaking-118300]]
+.Fails migrations for unknown types
+[%collapsible]
+====
+*Details* +
+Unknown saved objects types now cause {kib} migrations to fail. For more information, refer to {kibana-pull}118300[#118300].
+
+*Impact* +
+To complete the migration, re enable plugins or delete documents from the index in the previous version.
+====
+
+[discrete]
+[[breaking-116821]]
+.Removes deprecated config fields from Logs and Metrics APIs and SavedObjects
+[%collapsible]
+====
+*Details* +
+On the Logs and Metrics UIs, references to the following API and saved object deprecated fields have been removed: 
+
+* `timestamp`
+* `tiebreaker`
+* `container`
+* `pod`
+* `host`
+
+For more information, refer to {kibana-pull}116821[#116821] and {kibana-pull}115874[#115874].
+
+*Impact* +
+When you upgrade to 8.0.0, you are unable to use references to the deprecated fields.
+====
+
+// end::notable-breaking-changes[]
+
+To review the breaking changes in previous versions, refer to the following: 
+
+<<breaking-changes-8.0.0-beta1,8.0.0-beta1>> | <<breaking-changes-8.0.0-alpha2,8.0.0-alpha2>> | 
+<<breaking-changes-8.0.0-alpha1,8.0.0-alpha1>>
+
+[float]
+[[deprecations-8.0.0-rc1]]
+=== Deprecations
+
+Deprecated functionality does not have an immediate impact on your application, but we strongly recommend you make the necessary updates after you complete the upgrade.
+
+[discrete]
+[[deprecation-120689]]
+.Renames the `autocreate` data view APM setting
+[%collapsible]
+====
+*Details* +
+The `xpack.apm.autocreateApmIndexPattern` APM setting has been removed. For more information, refer to {kibana-pull}120689[#120689].
+
+*Impact* +
+To automatically create data views in APM, use `xpack.apm.autoCreateApmDataView`.
+====
+      
+[discrete]
+[[deprecation-119494]]
+.Updates Fleet API responses for consistency
+[%collapsible]
+====
+*Details* +
+To make sure all Fleet API GET resposes return `items`, the following have been updated:
```

Comment:
This description doesn't really capture everything that has changed (as described in the impact section). Plus I don't see any detail about `/api/fleet/agents` in under "impact." It feels like the "Details" section is too specific and the "Impact" section is incomplete.

Maybe someone on the dev team should clarify based on what we want to highlight for users. I don't think I'm the person to make this call.

Reply:
@dedemorton can you add someone from the Dev team to review?

Reply:
I think @juliaElastic is probably the best person to comment since they opened https://github.com/elastic/kibana/pull/119494.

Reply:
@dedemorton hi, is the CHANGELOG generated from somewhere or are you expecting me to suggest how to update it?

Reply:
@juliaElastic I think these entries begin in the changelog, but get edited and formatted by the Kibana writers. You should suggest changes if you think (as I do) that this section needs more work.

Reply:
@dedemorton @juliaElastic I need to merge these release notes in preparation for the release tomorrow. Please feel free to open a separate PR for any changes you want to make, and let me know if you need any help.

Reply:
@dedemorton 
So to summarize: 
Changes in Fleet API for consistency, previous versions are still working, and deprecated.

1. Rename hyphens to undescores:
- /api/fleet/enrollment-api-keys
- /api/fleet/agent-status
- /api/fleet/service-tokens
2. Change packages endpoint to split `pkgkey` path parameter:
- /epm/packages/{pkgkey} to /epm/packages/{packageName}/{version}
3. Rename `response` or `list` to `items` in response properties:
-  /api/fleet/enrollment_api_keys
- /api/fleet/agents
- /epm/packages/
- /epm/categories
- /epm/packages/_bulk
- /epm/packages/limited
- POST and DELETE /epm/packages/{packageName}/{version}
4. Rename `response` to `item` in response properties:
- GET and PUT /epm/packages/{packageName}/{version}